cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -425,7 +427,7 @@ impl Node<Option<cst::Policy>> {
                         slot.clone(),
                         if is_when { "when" } else { "unless" },
                     ),
-                    slot.loc.unwrap_or_else(|| c.loc.clone()),
+                    slot.loc.or_else(|| self.loc.clone()),
```

Comment:
intentional change from `c.loc` to `self.loc`?

Reply:
Not really, good catch.